src/dashboards/components/DashboardHeader.tsx

```diff
@@ -180,15 +191,39 @@ const DashboardHeader: FC<Props> = ({
         </Page.ControlBarLeft>
         <Page.ControlBarRight>
           <TimeZoneDropdown />
+          <TimeRangeDropdown
+            onSetTimeRange={handleChooseTimeRange}
+            timeRange={timeRange}
+          />
           <AutoRefreshDropdown
             onChoose={handleChooseAutoRefresh}
             onManualRefresh={resetCacheAndRefresh}
             selected={autoRefresh}
+            showAutoRefresh={false}
           />
-          <TimeRangeDropdown
-            onSetTimeRange={handleChooseTimeRange}
-            timeRange={timeRange}
-          />
+          {isFlagEnabled('new-auto-refresh') && (
+            <Button
+              text={
+                autoRefresh?.status &&
+                autoRefresh.status === AutoRefreshStatus.Active
+                  ? 'Stop Auto Refresh'
+                  : 'Enable Auto Refresh'
+              }
+              color={
+                autoRefresh?.status &&
+                autoRefresh.status === AutoRefreshStatus.Active
+                  ? ComponentColor.Danger
+                  : ComponentColor.Primary
+              }
+              onClick={
+                autoRefresh?.status &&
+                autoRefresh.status === AutoRefreshStatus.Active
```

Comment:
We should probably set an ‘isActive’ const somewhere instead  of doing this 3 times